interfaces/builtin/steam_support.go

```diff
@@ -399,7 +399,7 @@ func init() {
 	registerIface(&steamSupportInterface{commonInterface{
 		name:                  "steam-support",
 		summary:               steamSupportSummary,
-		implicitOnCore:        false,
+		implicitOnCore:        true,
```

Comment:
shouldn't this be conditional on the presence of something? I'm slightly uncomfortable having this slot present when it doesn't make sense

Reply:
We could use `release.OnCoreDesktop` here, but it's not really depending on anything specific in Core Desktop's base. The additional infrastructure the Steam snap needs is provided by other snaps (something to provide `x11`, `wayland`, and `desktop` slots).

The interface is primarily about allowing Steam's pressure-vessel sandbox system to function, so is mostly mount namespace manipulation.